developer-guides/rest-api/chat/ignoreuser/README.md

```diff
@@ -0,0 +1,37 @@
+# Ignore user
+
+Ignores an user in a chat.
+
+| URL | Requires Auth | HTTP Method |
+| :--- | :--- | :--- |
+| `/api/v1/chat.ignoreUser` | `yes` | `GET` |
+
+## Payload
+
+| Argument | Example | Required | Description |
+| :--- | :--- | :--- | :--- |
+| `rid` | `KLTM332QEwWLPJrjRPtcPHimPnLHsusiDF` | Required | The room ID. |
+| `userId` | `7aDSXtjMA3KPLxLjt` | Required | The User ID. |
+
```

Comment:
I think that is missing the `ignore` query parameter, as we can see [here](https://github.com/RocketChat/Rocket.Chat/blob/develop/app/api/server/v1/chat.js#L331).